### ISSUE TITLE
New Afterglow Prismatic Wired Xbox One Controller Not supported

### ISSUE THREAD

**Issue by Liljonny11** [paroj/xpad#128]
It appears PDP updated their Prismatic AfterGlow controller in 2019, I actually had the old one (the ones with button wheels on the back) and it worked fine. It seems PDP changed the way the driver behaves because it behaves very differently even on Windows, even their official PDP app is yet to be supported for the new controller. I'm a software engineer but don't know much about xpad, I took a look at xpad.c and noticed that  the string for the name of the prismatic controller has changed, it seems despite the Prismatic lights, it's not considered "Prismatic" when it comes to device description. "Afterglow Wired Controller for Xbox One" is the string pulled up in device description usb devices file. Tried adding the controller but couldn't  get it to work, any help would be great since the controller is useless on linux. 

I can post the sys/kernel/debug/usb/devices entry for the controller if that helps with implementation.

**Comment by cgutman** [comment]
Yes, please post `lsusb -v` output for your gamepad. I'm curious to see if it falls into the quirk already created for PDP gamepads.
